lib/fined.js

```diff
@@ -0,0 +1,141 @@
+const fs = require('fs');
+const path = require('path');
+
+const isArray = require('lodash.isarray');
+const isString = require('lodash.isstring');
+const isPlainObject = require('lodash.isplainobject');
+const isEmpty = require('lodash.isempty');
+const mapValues = require('lodash.mapvalues');
+const pick = require('lodash.pick');
+
+const expandTilde = require('expand-tilde');
+const parsePath = require('parse-filepath');
+
+
+function search(options, extensions, cwd) {
```

Comment:
No need to be sorry @sttk, I am **so** appreciative of the work you are doing, thank you again!